source/_components/rollershutter.command.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Command line Rollershutter"
+description: "Instructions how to have rollershutters call command line commands."
+date: 2016-02-22 13:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: command_line.png
```

Comment:
it is my intention, if you agree, to add the same icon to all the command line platforms.